radicle-cli/src/commands.rs

```diff
@@ -14,6 +14,8 @@ pub mod rad_delegate;
 pub mod rad_edit;
 #[path = "commands/help.rs"]
 pub mod rad_help;
+#[path = "commands/identity.rs"]
+pub mod rad_identity;
```

Comment:
I'd rename the file and module to use `id`, we try to match them exactly to the command name